### PR TITLE
Feat/overthebox improvements

### PR DESCRIPTION
**Pull request by Steffy29**
| Question         | Answer
| ---------------- | ---
| Branch?          | `develop`
| Bug fix?         | no
| New feature?     | no
| Breaking change? | no
| Tickets          | Fix #UXCT-305
| License          | BSD 3-Clause

## Description

Refactoring some part of code :
* use routing part in order to avoid relying $state inside a controller
* using the default $ctrl

### CODE REVIEW

**Review by lizardK — APPROVED** [review]

File: packages/manager/modules/overthebox/src/overthebox/configure/overTheBox-configure.component.js [hunk 1]
```diff
@@ -4,4 +4,8 @@ import template from './overTheBox-configure.html';
 export default {
   controller,
   template,
+  bindings: {
+    orderHash: '<',
```
Comment (JeremyDec) [inline packages/manager/modules/overthebox/src/overthebox/configure/overTheBox-configure.component.js:8]: Hmm this doesn't seems to be used no?

Comment (Steffy29) [reply packages/manager/modules/overthebox/src/overthebox/configure/overTheBox-configure.component.js:8]: No it's used into the file overTheBox-configure.html

Comment (frenautvh) [reply packages/manager/modules/overthebox/src/overthebox/configure/overTheBox-configure.component.js:8]: i don't see either where it's used can you link the line ?

Comment (Steffy29) [reply packages/manager/modules/overthebox/src/overthebox/configure/overTheBox-configure.component.js:8]: L43 of overTheBox-configure.html (which not change)

**Review by cbourgois — DISMISSED** [review]

**Review by marie-j — APPROVED** [review]

**Review by antleblanc — DISMISSED** [review]

**Review by JeremyDec — DISMISSED** [review]

**Review by antleblanc — APPROVED** [review]

**Pull request merged**